Source/WebCore/rendering/RenderBlockFlow.h

```diff
@@ -551,6 +551,8 @@ class RenderBlockFlow : public RenderBlock {
     // FIXME: This is temporary until after we remove the forced "line layout codepath" invalidation.
     std::optional<LayoutUnit> m_previousModernLineLayoutContentBoxLogicalHeight;
 
+    Vector<RenderLayer*> m_childLayersNeedingTransformUpdate;
```

Comment:
This violates our "no new raw pointers" rule. This needs to use weak pointers.

Reply:
Right fixed.